cargo-dist/src/backend/installer/homebrew.rs

```diff
@@ -49,6 +51,40 @@ pub(crate) fn write_homebrew_formula(
 ) -> DistResult<()> {
     let mut info = source_info.clone();
 
+    // Collect all dist-manifests and fetch the appropriate Mac ones
+    let mut manifests = vec![];
+    for file in glob(&format!("{}/*-dist-manifest.json", &graph.dist_dir))? {
```

Comment:
I used a glob for convenience, but given the simplicity of the naming format, we can probably forego the extra dependency and manually iterate every file in that directory to check if it fits our naming format.

Reply:
Right now the code to parse and instantiate the dist manifests lives here, but it doesn't have to stay that way. If we anticipate using it elsewhere, it may be better to have it in a helper function somewhere.